CryptoPrices/Sources/Styleguide/Sources/Styleguide/CommonViews/CircularProgressView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ActivityRingView.swift
+//  Styleguide
+//
+//  Created by Doan Thieu on 18/01/2023.
+//
+
+import SwiftUI
+
+public struct CircularProgressView: View {
+
+    private let progress: Double
+    private let tint: Color
+
+    public var body: some View {
+        Circle()
+            .trim(from: 0.0, to: progress)
+            .stroke(tint, lineWidth: 4.0)
+            .frame(width: 48.0, height: 48.0)
+    }
+
+    public init(progress: Double, tint: Color = .gray) {
```

Comment:
It seems we don't use the default (.gray) color anywhere. Do you think we should replace the default color `.gray` with `Colors.tiffanyBlue.swiftUIColor`?